src/admin/content-block/components/wrappers/MediaGridWrapper/MediaGridWrapper.tsx

```diff
@@ -147,7 +147,7 @@ const MediaGridWrapper: FunctionComponent<MediaGridWrapperProps> = ({
 				ctaWidth={ctaWidth}
 				ctaButtonAction={ctaButtonAction}
 				elements={(resolvedResults || []).map(mapCollectionOrItemData)}
-				navigate={buttonAction =>
+				navigate={(buttonAction: any) =>
```

Comment:
Misschien is dit het moment om dit type eens aan te maken, want we hadden dit ook besproken tijdens onze content-picker-debug sessie. Maar "ButtonAction" is niet algemeen genoeg aangezien dit ook meegegeven kan worden aan een video als item.

Reply:
Mss ContentLink? We doen dit dan best als aparte PR